tests/network-tests/src/flows/carthage/constantAuthorities.ts

```diff
@@ -0,0 +1,22 @@
+import { extendDebug } from 'src/Debugger'
+import { FlowProps } from 'src/Flow'
+import { expect, assert } from 'chai'
+
+export default async function constantAuthorities({ api, query, env }: FlowProps): Promise<void> {
+  const debug = extendDebug('flow: constant Authorities in PoA')
+  debug('started')
+  api.enableDebugTxLogs()
+
+  const forceEra = await api.getForceEra()
+  assert(forceEra.isForceNone)
+
+  // babe authorities
+  const currentAuthorities = await api.getBabeAuthorities()
+  const nextAuthorities = await api.getNextBabeAuthorities()
+  expect(nextAuthorities).to.be.deep.equal(currentAuthorities)
+
+  // make sure that next Session keys are none
```

Comment:
the comment seems misleading, don't we mean next session keys are the same as current authority session keys?

Reply:
Technically the rule is : next session keys None -> keep the current keys set as the next one